include/openthread/link.h

```diff
@@ -788,8 +788,13 @@ void otLinkFilterClearAddresses(otInstance *aInstance);
 otError otLinkFilterGetNextAddress(otInstance *aInstance, otMacFilterIterator *aIterator, otMacFilterEntry *aEntry);
 
 /**
- * Adds a fixed received signal strength (in dBm) entry for the messages from a given Extended Address in
- * MAC Filter.
+ * Adds the specified Extended Address to the `RssIn` list (or modifies an existing
+ * address in the `RssIn` list) and sets the
+ * received signal strength (in dBm) entry for messages from that address.
+ * The Extended Address does not necessarily have to be in the `address allowlist/denylist` filter
+ * to set the `rss`.
```

Comment:
Align the lines to wrap around same length? Typically  we wrap at around 120 char for the method docs, or some folks prefer ~80 chars.
```suggestion
 * Adds the specified Extended Address to the `RssIn` list (or modifies an existing
 * address in the `RssIn` list) and sets the received signal strength (in dBm) entry 
 * for messages from that address. The Extended Address does not necessarily have
 * to be in the `address allowlist/denylist` filter  to set the `rss`.
```